### ISSUE TITLE
Define a set of labels we want to use

### ISSUE THREAD

**Issue by camilleriluke** [camilleriluke/ejja-ha-nieklu#28]
These are the initial set of labels I came up with. The idea is to have label in the following form `category:subcategory`

### Indicate which area the issue is related to
- Area: Backend
- Area: Doc
- Area: Frontend

### Indicate the current status of a the issue
- Status: In Pull Reuqest
- Status: Needs Discussion

### Indicate the type of the issue
- Type: Bug
- Type: Enhancment
- Type: Feature
- Type: Idea
- Type: Invalid
- Type: Question
- Type: Refactor

Basically any issue will hold at least on of each category.

Any other suggestions?

**Comment by KPull** [comment]
No one can assign labels to issues though right now, no?

**Comment by camilleriluke** [comment]
I'll keep the labels in sync, the idea behind labels is to make it easy for contributors to filter issues.
